proto/cometbft/version/v1/types.proto

```
@@ -0,0 +1,24 @@
+syntax = "proto3";
+package cometbft.version.v1;
+
+option go_package = "github.com/cometbft/cometbft/api/cometbft/version/v1";
+
+import "gogoproto/gogo.proto";
```

Buf (1.54.0)

import "gogoproto/gogo.proto": file does not exist

(COMPILE)